### DISCUSSION TITLE
Strive Score

### DISCUSSION THREAD

**Discussion by GregChung-IBFL** [philosowaffle/peloton-to-garmin#108] [Ideas]
The new Strive Score thing is showing up in the workoutSamples as "effort_zones".  Is there any analogy for an activity rating or other custom metric in the Garmin?  At worst, I suppose it can be copied over to the activity comments?

**Comment by philosowaffle** [comment]
Hmmmm, nothing immediately comes to mind, I'll have to do some research.

**Comment by philosowaffle** [comment]
I did a ride today that had [Strive Score](https://support.onepeloton.com/hc/en-us/articles/360059490452-Strive-Score) and was able to read more about it. Seems this is just a multiplier that correlates to time spent in each Heart Rate zone.  This is likely Peloton's own version of [Garmin's Intensity Minutes](https://support.garmin.com/en-US/?faq=pNU9nnDzzGAHmEavp9rpY8) and [Stravas Suffer Score/Training Impulse](https://blog.strava.com/suffer-score-how-hard-is-hard-11775/). All of which boil down to:

```
Time in each HR zone + platform specific magic sauce = Platform Specific 'Score' number
```

Currently, I'm not thinking it adds any value to track this number when importing to Garmin since its not a metric that will make sense on the Garmin platform.  It can't be compared to Garmin Intensity minutes, and Intensity Minutes is not something we set, it is calculated by Garmin when the workout is uploaded.

**Reply by GregChung-IBFL** [reply]
I noticed something new w/ the Strive Score today.  There was another person in the same class "session" who also had a Strive Score.  Moreover, there was a color coded bar to the right of both our names in the leaderboard, presumably reflecting the Strive Score.  It was interesting because to see any other leaderboards stats, except for total output, you have to click into the specific name.

**Reply by philosowaffle** [reply]
I noticed that too. When I clicked on the person's name to see the detailed stats view, I noticed the display added the HR zone info next to the color bar.  My theory is the color stripe on the LB reflects the persons current HR zone (and by extension, effort) and does not match the strive score. But that is just a theory.
